### ISSUE TITLE
smart cable for Fartshark

### ISSUE THREAD

**Issue by jonathan2018** [JyeSmith/18650-Backpack#2]
this is project can modifi to  smart cable for Fartshark ? 
Feature  : 
ON/OFF  power 
display volt ampe when Fs work 
alarm batterry ,auto Off when not use a long time 
thanks

**Comment by JyeSmith** [comment]
You sure can.  Just stick it on a cable with an inline power switch.

But at the moment it only takes a 2s battery.  This is due to the step up 9v regulator powering the goggles.  If this was replace with a S10V3F9 then you could use up to 4s, but I have not tested this.

Pololu 9V Step-Up/Step-Down Voltage Regulator S10V3F9
https://www.pololu.com/product/2095

**Comment by jonathan2018** [comment]
Thanks for answer 
I rebuild diagram for project with button power  by Pin SCK  attiny85. However , attiny85 with 4k flash maybe no problem ?

**Comment by JyeSmith** [comment]
Yep the flash is pretty much full at the moment.  You'll need to use a non software on off button.

**Comment by jonathan2018** [comment]
i m reading source code to understand and modifi from your code . maybe lost times for me to do it work

**Comment by JyeSmith** [comment]
Any reason you need a software power button?

**Comment by jonathan2018** [comment]
reason : i want to ON/OFF power for FS  instead plug in or remove Jack .Fs have not Switch power .
Display infor volt ,ampe ...on LCD  is advantage and combined  feature sofware power is so good.

Orginal your project have all infor on LCD OLED  .

**Comment by skaman82** [comment]
Great project: made something similar https://github.com/skaman82/PowerPuck :)

**Comment by JyeSmith** [comment]
@skaman82 I came across yours when I started mine... This is how I found the 9v reg to use 😉

By the way, your prints look awesome!!! I just heatshrinked mine and have it dangling from my head LOL
